src/PropTree.ts

```diff
@@ -36,40 +33,35 @@ export class PropTree
         }
 
         // build the tree of props
-        for (let i = 0; i < props.length; ++i)
-        {
+        for (let i = 0; i < props.length; ++i) {
             const prop = props[i];
 
-            if (!prop || !prop.name)
-                continue;
+            if (!prop || !prop.name) continue;
 
             const parts = prop.name.split('.');
 
             const obj = this.nodes[prop.name];
 
-            if (!obj)
-            {
-                warn('Failed to find dot-notation property in map. This is likely a bug.');
+            if (!obj) {
+                warn(
+                    'Failed to find dot-notation property in map. This is likely a bug.',
+                );
```

Comment:
This seems like a weird change.

Reply:
This is probably because of the line length that it gets wrapped.